batch/batch/server/google_storage.py

```diff
@@ -0,0 +1,25 @@
+from google.cloud import storage
+
+
+gcs_client = storage.Client()
+
+
+def upload_private_gs_file_from_string(bucket, target_path, string):
+    bucket = gcs_client.bucket(bucket)
+    f = bucket.blob(target_path)
+    f.metadata = {'Cache-Control': 'no-cache'}
+    f.upload_from_string(string)
+
+
+def download_gs_file_as_string(bucket, path):
+    bucket = gcs_client.bucket(bucket)
+    f = bucket.blob(path)
+    f.metadata = {'Cache-Control': 'no-cache'}
```

Comment:
@danking Is this the function you used to make it asynchronous?

```
async def blocking_to_async(f, *args, **kwargs):
     return await asyncio.get_event_loop().run_in_executor(
         app['blocking_pool'], lambda: f(*args, **kwargs))
```

If so, I think I should wait and rebase on your PR.